python/astro_metadata_translator/headers.py

```diff
@@ -11,17 +11,22 @@
 
 """Code to support header manipulation operations."""
 
-__all__ = ("merge_headers",)
+__all__ = ("merge_headers", "fix_header")
 
 import logging
 import itertools
 import copy
+import os
+import yaml
 
 from .translator import MetadataTranslator
 from .translators import FitsTranslator
 
 log = logging.getLogger(__name__)
 
+ENV_VAR_NAME = "METADATA_CORRECTIONS_PATH"
+"""Name of environment variable containing search path for header fix up."""
+
```

Comment:
Why is this set as an environment variable? I was expecting it to be in a known place relative to `obs_lsst`

Reply:
There are three ways to find correction files:

1. Pass in explicit search path.
2. Read a path from an environment variable.
3. Ask the translator class for explicit searchpaths.

Option 3 is how LSST works. See the lsst/obs_lsst#97 PR where I add a `search_paths` method that specified `$OBS_LSST_DIR/corrections` be used for the path.